modules/net-lb-app-ext-regional/variables-backend-service.tf

```diff
@@ -31,8 +31,7 @@ variable "backend_service_configs" {
     session_affinity                = optional(string)
     timeout_sec                     = optional(number)
     backends = list(object({
-      # group renamed to backend
-      backend         = string
+      group           = string
```

Comment:
I would not change this :)

Reply:
The names are not consistent across modules. 

There are missing properties across modules as well. Can you elaborate why standardizing on 1 name is not advised?

Reply:
Some properties are missing because different load balancera support different features.

I would not touch those modules, they are very complex and delicate and have worked well for a long time.